runtime/frequency/src/lib.rs

```diff
@@ -170,7 +172,11 @@ pub type UncheckedExtrinsic =
 pub type CheckedExtrinsic = generic::CheckedExtrinsic<AccountId, RuntimeCall, SignedExtra>;
 
 /// Migrations for Frequency
-pub type Migrations = (remove_sudo::RemoveSudo, pallet_msa::migration::Migration<Runtime>);
+pub type Migrations = (
+	remove_sudo::RemoveSudo,
+	pallet_msa::migration::Migration<Runtime>,
+	pallet_schemas::migrations::SchemaMigrationToV1<Runtime>,
```

Comment:
Might want to follow the pallet_msa pattern for single migration block that calls the versioned ones.

Reply:
@saraswatpuneet you might know better about this one

Reply:
It is very similar, just the naming is such, I will just name it to `SchemaMigration`